setup.py

```diff
@@ -37,8 +37,12 @@ def get_version():
 setup(
     name='sphinxcontrib-websupport',
     version=get_version(),
-    url='http://sphinx-doc.org/',
-    download_url='https://pypi.org/project/sphinxcontrib-websupport/',
+    project_urls={
+        'Homepage': 'https://sphinx-doc.org/',
+        'Download': 'https://pypi.org/project/sphinxcontrib-websupport/',
+        'Source': 'https://github.com/sphinx-doc/sphinxcontrib-websupport',
+        'Conda-Forge': 'https://github.com/conda-forge/sphinxcontrib-websupport-feedstock',
```

Comment:
What is Conda-Forge?
Sphinx maintainers are not involved in that repository. IMO, this URL cannot be added.
It can be merged once this line is removed.

Reply:
Hi! Conda-Forge is a repository similar in spirit to Debian that redistributes open-source software (in accordance with the license, of course). Packages are build to be installed into a "Conda environment" which is the same idea as a Python venv, but it works with and manages dependencies for non-Python software as well.

As requested, I have removed the `Conda-Forge` line.